Program.cs

```diff
@@ -1,7 +1,37 @@
 ﻿namespace Training {
     internal class Program {
         static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
+         string[] words = File.ReadAllLines ("C:\\Users\\priyanka.s\\Downloads\\words.txt");
+         char[] letters = { 'U', 'X', 'L', 'T', 'A', 'E', 'N' };
+         int total = 0, total2 = 0, total1 = 0, index = 0; ;
+         string[] wordarray = new string[words.Length];
+         int[] point = new int[words.Length];
+         foreach (string word in words) {

```

Comment:
If you logic requires index, you could use for loop instead of foreach.

Reply:
Changes are done